G3Common/src/users/BranchManager.java

```diff
@@ -52,4 +51,11 @@ public void setBranchName(Branch branchName) {
 		this.branchName = branchName;
 	}
 
+	/**
+	 * This is the toString for this class
+	 */
+	@Override
+	public String toString() {
+		return "Branch manager: "  + userFirstName + " " + userLastName + ", user ID = " + userId;
```

Comment:
I am not sure that we need to enter here his ID, I would enter the branch name instead. 
I suggest: return branchName + "manager: " + userFirstName + " " + userLastName;

Reply:
fixed